benchmarks/test_sigmaM.py

```diff
@@ -32,6 +32,6 @@ def test_sigmaM(model, w0, wa):
 
     for i in range(data.shape[0]):
         m = data[i, 0] / cosmo['h']
-        sm = ccl.sigmaM(cosmo, m, 1)
+        sm = ccl.halos.sigmaM(cosmo, m, 1)
```

Comment:
This is an API break. I am ok with that, but it means we are going to be shipping v3 on master.

Reply:
I'm OK with this too, but I'm also happy to move `sigmaM` out of `halos`

Reply:
Hmmmmmm. As long as we don't create a circular import, keeping it out of halos actually makes a ton of sense. I'd vote to keep sigmaM where it is and not move it into halos.

Reply:
Is this just because of the API break? Because we're gonna be breaking it anyway

Reply:
No, I like it more aesthetically, but I am ok either way.

Reply:
ok, we could put this into power.py (that's where `sigmaR` lives)

Reply:
Yup. Let's do that.

Reply:
Done